Subject: Embed more programs in the initramfs/initrd
Author: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Date:  2020-05-21
Bug: https://github.com/latchset/clevis/issues/195
Last-Update: 2020-11-22

--- a/src/initramfs-tools/hooks/clevis.in
+++ b/src/initramfs-tools/hooks/clevis.in
@@ -62,6 +62,7 @@
 copy_exec @bindir@/clevis-decrypt || die 1 "@bindir@/clevis-decrypt not found"
 copy_exec @bindir@/clevis-luks-common-functions || die 1 "@bindir@/clevis-luks-common-functions not found"
 copy_exec @bindir@/clevis-luks-list || die 1 "@bindir@/clevis-luks-list not found"
+copy_exec @bindir@/clevis-luks-unlock || die 1 "@bindir@/clevis-luks-unlock not found"
 if [ -x @bindir@/clevis-decrypt-tpm2 ]; then
     copy_exec @bindir@/clevis-decrypt-tpm2 || die 1 "@bindir@/clevis-decrypt-tpm2 not found"
     tpm2_creatprimary_bin=$(find_binary "tpm2_createprimary")
--- a/src/luks/dracut/clevis/module-setup.sh.in
+++ b/src/luks/dracut/clevis/module-setup.sh.in
@@ -51,6 +51,7 @@
         grep sed cut \
         clevis-decrypt \
         clevis-luks-list \
+        clevis-luks-unlock \
         cryptsetup \
         luksmeta \
         clevis \
